src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
```

Comment:
According to the code style rules, there should be an empty line between the block styles and nested selectors. Please add an empty line before '&:hover' and '&__image' for better readability.